src/rest/v1/openapi.yml

```diff
@@ -127,6 +129,39 @@ components:
           type: string
       required:
         - shiftUuid
+    RateWorkerInput:
+      type: object
+      properties:
+        shiftUuid:
+          type: string
+        workerUuid:
+          type: string
+        rating:
```

Comment:
Add min and max restrictions for rating

Reply:
Didn't know how to do that here but caught it in the DB and service level validation.